### PR TITLE
Format dicom

### PR DESCRIPTION
**Pull request by tcpan**
Initial DICOM benchmark.  Please note that the current generated DICOM file is not YET dicom standards compliant, and the tooling has not yet been optimized for random data access.

### CODE REVIEW

File: waveform_benchmark/formats/dicom.py [hunk 1]
```diff
@@ -0,0 +1,626 @@
...
+        dicom = self.make_empty_dcm_dataset("genericECG")
+        dicom = self.set_order_information(dicom)
+        dicom = self.set_study_information(dicom)
+        dicom = self.set_waveform_acquisition_info(dicom)
+        dicom = self.add_waveform_chunks_multiplexed(dicom, waveforms)        
+        
+        # Save DICOM file.  write_like_original is required
+        dicom.save_as(path, write_like_original=False)
+        
+        dicom.save_as("/mnt/c/Users/tcp19/Downloads/Compressed/test_waveform.dcm", write_like_original=False)
```
Comment (briangow) [inline waveform_benchmark/formats/dicom.py:484]: I think this line needs to be removed.

File: waveform_benchmark/formats/test.py [hunk 2]
```diff
@@ -0,0 +1,267 @@
+# %%
```
Comment (briangow) [inline waveform_benchmark/formats/test.py:1]: Did you mean to include this `formats/test.py` file in your pull request?

**Review by briangow — CHANGES_REQUESTED** [review]

File: waveform_benchmark/formats/dicom.py (hunk 1 above)
Comment (tcpan) [reply waveform_benchmark/formats/dicom.py:484]: yep . sorry about that.

File: waveform_benchmark/formats/test.py (hunk 2 above)
Comment (tcpan) [reply waveform_benchmark/formats/test.py:1]: also an oversight.  sorry.

**Review by briangow — APPROVED** [review]
Thanks for the updates! I'm going to approve this. Feel free to merge if you aren't planning any other immediate changes.

**Pull request merged**